config/routes.rb

```diff
@@ -9,3 +9,7 @@
   # http://guides.rubyonrails.org/routing.html
   root to: 'home#index'
 end
+
+Rails.application.routes.draw do
```

Comment:
You should be able to drop lines 11 and 13 so that your new pages route is in the existing `Rails.application.routes.draw` block.